src/main/scala/org/zalando/spearheads/innkeeper/services/RoutesService.scala

```diff
@@ -106,6 +110,14 @@ class DefaultRoutesService @Inject() (
     }
   }
 
+  override def allDeletedRoutes: Source[RouteOut, NotUsed] = {
```

Comment:
Actually, I think that it is not needed to have a separate test for the service. It does not do anything.  This code is tested during acceptance tests.